### ISSUE TITLE
Add Kokoro 82M TTS for offline multilingual voice support

### ISSUE THREAD

**Issue by mattmezza** [mattmezza/humux#84]
## Motivation

The current voice pipeline uses edge-tts (MS Edges cloud TTS) for speech synthesis. This requires an internet connection, ties the agent to a closed-source provider, and limits voice control. Adding **Kokoro 82M** — an open-weight, on-device TTS model — would give the agent:

- **Fully offline voice synthesis** — no API calls, no internet dependency, zero latency from network round-trips
- **Multilingual voices** — Kokoro 82M supports multiple languages out of the box (English, Japanese, Korean, French, Mandarin, Spanish, etc.), enabling the agent to speak in the same language its conversing in (no more mixing Italian thoughts with English TTS)
- **Cool factor** — its a single ~82M parameter model that runs on CPU in real-time, so no GPU needed

### 1. Add Kokoro as a TTS backend option

The `voice/pipeline.py` module currently has a `synthesize()` method that calls edge-tts. Add a Kokoro backend that:

```python
class KokoroTTS:
    def __init__(self, model_path: str = "models/kokoro/kokoro-v0_19.onnx", voices_dir: str = "models/kokoro/voices/"):
        self.model = load_kokoro(model_path)
        self.voices = load_voices(voices_dir)

    async def synthesize(self, text: str, voice: str = "af_bella") -> bytes:
        audio = self.model.generate(text, voice=self.voices[voice])
        return audio.tobytes()
```

### 2. Voice configuration

Extend the `voice` section in config to support multiple backends:

```yaml
voice:
  backend: "kokoro"           # "edge-tts" | "kokoro"
  tts_enabled: true
  kokoro:
    model_path: "models/kokoro/kokoro-v0_19.onnx"
    voices_dir: "models/kokoro/voices/"
    default_voice: "af_bella" # English female, multi-style
```

Available Kokoro voices include:
- `af_bella`, `af_nicole`, `af_sky` — English female
- `am_adam`, `am_michael` — English male
- `bf_emma`, `bf_isabella` — English (British) female
- `bm_george`, `bm_lewis` — English (British) male
- `ff_siwis` — French female
- `hm_azaria` — Hindi male  
- `jf_alpha`, `jm_kuma` — Japanese female/male
- `kf_alpha`, `km_jisu` — Korean female/male
- `mf_alpha`, `mm_alpha` — Mandarin female/male
- `pf_dora`, `pm_alex` — Portuguese female/male
- `ef_dora`, `em_alex` — Spanish female/male

### 3. Model bundling

The Kokoro ONNX model (~170MB) and voice embeddings should be **prefetched at Docker build time** (like the Whisper model) so the container works offline from the first run:

```dockerfile
# Download Kokoro model
RUN mkdir -p /app/models/kokoro && \
    curl -sSL https://github.com/.../kokoro-v0_19.onnx -o /app/models/kokoro/kokoro-v0_19.onnx && \
    curl -sSL https://github.com/.../voices.json -o /app/models/kokoro/voices.json
```

### 4. Per-voice personas

Allow the persona YAML frontmatter to specify a Kokoro voice:

```yaml
character: |
  ...
voice: "af_bella"    # used when this persona is active
```

This way the coding helper can speak in one voice and the language tutor in another (e.g. a French voice when teaching French).

### 5. Fallback

Keep edge-tts as the default/fallback backend. If Kokoro fails to load (missing model, OOM), fall back gracefully to edge-tts with a log warning.

## Out of scope (first iteration)

- Streaming/real-time chunked audio — Kokoro generates the full audio at once
- SSML support — Kokoro uses plain text
- Voice cloning/fine-tuning — pre-trained voices only

## Dependencies

- Kokoro ONNX model: https://huggingface.co/hexgrad/Kokoro-82M
- Python binding: `pip install kokoro-onnx` (or similar — check latest packaging)

/label new
/label todo
